### ISSUE TITLE
Error bundling dynamic import + JSX

### ISSUE THREAD

**Issue by nikku** [goto-bus-stop/split-require#10]
I'm having trouble getting `split-require` with a ES2015 + JSX project to work ([cf. minimal example project](https://github.com/nikku/split-require-jsx-text)). It fails bundling dynamic import + JSX code.

I was not able to track down the issue to the babel transform for dynamic import or split-require. Both seem to work fine alone but fail when being combined.
  
__Test__

```javascript
import('./external').then(function(result) {
  console.log(result);
});


var app = <div></div>;
```

__Error__

```bash
❯ npm test
> @ test ~/Projects/split-require-jsx-text
> browserify index.js -t [ babelify ] -p [ split-require --dir dist ] -o dist/index.js
~/Projects/split-require-jsx-text/node_modules/split-require/plugin.js:327
    var requirePath = node.arguments[0].arguments[0].value
                                                 ^
TypeError: Cannot read property '0' of undefined
    at processSplitRequire (~/Projects/split-require-jsx-text/node_modules/split-require/plugin.js:327:50)
    at ~/Projects/split-require-jsx-text/node_modules/split-require/plugin.js:144:11
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:82:5)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:78:9)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:74:13)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:78:9)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:74:13)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:78:9)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:78:9)
    at walk (~/Projects/split-require-jsx-text/node_modules/transform-ast/index.js:78:9)
npm ERR! Test failed.  See above for more details.
```

**Comment by goto-bus-stop** [comment]
thanks for the repro!

the split-require plugin adds a transform that finds split-require calls and adds a real `require()` call around it so that browserify picks it up and finds the dependencies. the issue is that the browserify cli runs plugins first, before adding transforms. so, the split-require plugin adds its transform before browserify adds babelify.

setting the split-require transform to `global: true` fixes it because browserify runs global transforms last. the transform is quite cheap, it doesn't parse files that don't contain split-require calls, so this should be fine…

(the best way from my pov to fix this would be to have a means to teach browserify that `splitRequire` also adds a dependency, but that might take some work.)

**Comment by goto-bus-stop** [comment]
should be fixed by 📦 2.0.3!

**Comment by nikku** [comment]
Thanks!
